osv/ecosystems.py

```diff
@@ -24,6 +24,7 @@
 from .third_party.univers.debian import Version as DebianVersion
 from .third_party.univers.gem import GemVersion
 
+from . import packagist_version
```

Comment:
Oh one more nit: Please move this to after line 30 to preserve the alphabetic ordering.